lib/spack/spack/cmd/install.py

```diff
@@ -48,6 +48,9 @@ def setup_parser(subparser):
         '-n', '--no-checksum', action='store_true', dest='no_checksum',
         help="Do not check packages against checksum")
     subparser.add_argument(
+        '-p', '--parallel', action='store_true', dest='build_parallel',
```

Comment:
Ah, yeah, it's a boolean flag; missed that. However, this now means that we have parallel makes running at the same time as each other. This usually doesn't work out as well as one would hope. Though it can be useful if we also pass `-l` to the `make` processes to limit the system load during the build.